frontend/packages/knative-plugin/console-extensions.json

```diff
@@ -594,6 +574,38 @@
       ]
     }
   },
+  {
+    "type": "dev-console.add/action",
+    "properties": {
+      "id": "create-serverless-function-samples",
+      "groupId": "serverless-function",
+      "href": "/samples/ns/:namespace?sampleType=Serverless function",
+      "label": "%knative-plugin~Samples%",
```

Comment:
Just an opinion on that: I don't think we should repeat "function" in each action.

Instead of that, I would recommend that you **_discuss_** with our PM to bring Samples into the Developer Catalog in the next release.